pkg/downloader/manager.go

```diff
@@ -400,6 +401,11 @@ Loop:
 			continue
 		}
 
+		// if repo is an OCI registry, continue
+		if strings.HasPrefix(dd.Repository, fmt.Sprintf("%s://", registry.OciProtocol)) {
```

Comment:
there are now 3 `if` clauses here. Can we optimize this to prevent 3 separate evaluations?

Reply:
I don't think we'll get around 3 evaluations code-wise (and the compiler should be able to optimise what can be optimised there), but we can perhaps clean it up... I'm not sure what'd be the best approach. You could do something like

```golang
specialPrefixes := map[string]string{"oci", "file"}

parts := strings.Split(dd.Repository, "://")
if _, ok := specialPrefixes[parts[0]]; ok || dd.Repository == "" {
    continue
}
```

but to me, that doesn't feel any cleaner.

We can also chunk it into a single if statement, but that won't reduce the number of evaluations.